flycheck.el

```diff
@@ -232,6 +232,7 @@ attention to case differences."
     ruby-rubylint
     ruby
     ruby-jruby
+    ruby-reek
```

Comment:
Why do you register this syntax checker after all other Ruby checkers?  That's also the reason why the test fails: it runs with the plain `ruby` checker which finds no errors of course.

How did you image this checker to be used in relation to the other ruby checkers?